velox/core/SimpleFunctionMetadata.h

```diff
@@ -666,15 +666,17 @@ class UDFHolder {
   DECLARE_METHOD_RESOLVER(callAscii_method_resolver, callAscii);
   DECLARE_METHOD_RESOLVER(initialize_method_resolver, initialize);
 
-  // Check which flavor of the call() method is provided by the UDF object. UDFs
-  // are required to provide at least one of the following methods:
+  // Check which flavor of the call()/callNullable()/callNullFree() method is
+  // provided by the UDF object. UDFs are required to provide at least one of
+  // the following methods:
   //
-  // - bool|void call(...)
-  // - bool|void callNullable(...)
-  // - bool|void callNullFree(...)
+  // - bool|void|Status call(...)
+  // - bool|void|Status callNullable(...)
+  // - bool|void|Status callNullFree(...)
   //
-  // Each of these methods can return either bool or void. Returning void means
-  // that the UDF is assumed never to return null values.
+  // Each of these methods can return bool, void or Status. Returning void
+  // means that the UDF is assumed never to return null values. Returning
+  // Status to hold success or error outcome of the function call.
```

Comment:
how do we have a function that may return status and occasionally nulls? If this is not supported we should add comment to clarify.

Reply:
@pedroerp, thanks for your comment! Currently, Status type implies result is not null. Just added some comments to clarify.